presto-hive/src/main/java/com/facebook/presto/hive/HiveSessionProperties.java

```diff
@@ -133,6 +134,11 @@ public HiveSessionProperties(HiveClientConfig hiveClientConfig, OrcFileWriterCon
                         "Ignore table bucketing to enable reading from unbucketed partitions",
                         hiveClientConfig.isIgnoreTableBucketing(),
                         false),
+                integerProperty(
+                        MIN_BUCKET_COUNT_TO_NOT_IGNORE_TABLE_BUCKETING,
+                        "Ignore table bucketing when table bucket count is less than this session property",
```

Comment:
nit: `Ignore table bucketing when the number of buckets is less than the value specified`